draft-ietf-gnap-core-protocol.md

```diff
@@ -2057,9 +2084,7 @@ with a multiple access token structure containing one access token.
 ]
 ~~~
 
-Each access token MAY be bound to different keys with different proofing mechanisms.
-
-The `manage` URI MUST NOT contain the access token `value`.
+The parameters of each access token are separate. For example, each access token MAY be bound to different keys with different proofing mechanisms.
```

Comment:
yes, although that example is probably a bit extreme

Reply:
added lighter examples